test/testdata/config_pipeline.baseline

```diff
@@ -0,0 +1,4 @@
+mkdir -p {OUTPUT}
+protoc --proto_path=test/fake-repos/gapi-core-proto/src/main/proto/ --proto_path=test/fake-repos/fake-proto --proto_path=MOCK_GRADLE_TASK_OUTPUT --include_imports --include_source_info -o {OUTPUT}/library-v1.desc test/fake-repos/fake-proto/fake.proto
+mkdir -p {OUTPUT}/library-v1-config-gen
+test/fake-repos/gapi-tools/gradlew -p test/fake-repos/gapi-tools runConfigGen -Pclargs=--descriptor_set={OUTPUT}/library-v1.desc,--output={OUTPUT}/library-v1-config-gen/library-v1_veneer.yaml
```

Comment:
Does this baseline file need to be updated for your latest change? (doing the mv command)